.github/workflows/macos.yml

```diff
@@ -46,11 +46,13 @@ jobs:
         uses: ammaraskar/gcc-problem-matcher@0.2.0
 
       - name: Build (${{ matrix.build-type }})
+        env:
+            CMAKE_PREFIX_PATH: ${{ matrix.qt-version-major == '6'}} && ${{ env.Qt6_Dir }} || ${{ env.Qt5_Dir }}
```

Comment:
Where does `$Qt6_Dir` / `$Qt5_Dir` come from? Nevertheless this can be removed.

Reply:
On Linux, I had to do this in order to point to the correct Qt installation. I had to change it to `Qt6_DIR`  , though (instead of  `..Dir`  ) Not sure why it doesn't fail on macOS.